.changeset/polite-dingos-train.md

```diff
@@ -0,0 +1,33 @@
+---
+'@backstage/plugin-kubernetes': minor
+---
+
+Added Pod logs components for Kubernetes plugin
+
+**BREAKING**: `kubernetesProxyApi` must be defined as an apis in the plugin to use k8s proxy features
+
+```
+...
+export const kubernetesPlugin = createPlugin({
+  id: 'kubernetes',
```

Comment:
Can we change the verbiage here to mention that this type of setup is required for other plugins (besides the ootb kubernetes plugin) to consume the logs "porcelain"?

Reply:
fixed!